cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -132,15 +132,21 @@ func (c *CStorVolumeReplicaController) cVREventHandler(operation common.QueueOpe
 			return string(apis.CVRStatusDeletionFailed), err
 		}
 		return "", nil
+	case common.QOpModify:
+		fallthrough
 	case common.QOpSync:
-		glog.Infof("Synchronizing CstorVolumeReplica status for volume %s", cVR.ObjectMeta.Name)
+		if operation == common.QOpModify {
```

Comment:
why we need 'if-else'? we can just print 'opcode' in a single log statement which covers both